clean_project/clean_file.py

```diff
@@ -26,7 +25,8 @@ def sanitize(line):
     :param line: string to clean
     :return: new line cleaned
     """
-    return re.sub(ur"([^\w\d\s]+|\b\w{1}\b)", ' ', line).encode("utf-8")
+    # I deleted all punctuation and string of one character to simplify the search and find the right result
```

Comment:
I appreciate the comment, but I expect something along the lines of:
`This regex deletes all the punctuation and the strings of length 1, in order to simplify the search and find the right result.`